app/src/org/commcare/android/analytics/TimedStatsTracker.java

```diff
@@ -0,0 +1,112 @@
+package org.commcare.android.analytics;
+
+import android.content.SharedPreferences;
+import android.util.Log;
+
+import org.commcare.android.javarosa.AndroidLogger;
+import org.commcare.dalvik.application.CommCareApp;
+import org.commcare.dalvik.application.CommCareApplication;
+import org.javarosa.core.services.Logger;
+import org.joda.time.DateTime;
```

Comment:
Many of these imports are unused. I think our plugin that auto removes these isn't around anymore :(